.azure-pipelines/azure-pipelines.yml

```diff
@@ -0,0 +1,67 @@
+# Python package
+# Create and test a Python package on multiple Python versions.
+# Add steps that analyze code, save the dist with the build record, publish to a PyPI-compatible index, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/languages/python
+
+trigger:
+- master
+
+pr: none
+
+jobs:
+
+
+- job: 'Build_Publish_AKS_Deploy_CLI_Extension'
+  pool:
+    vmImage: 'Ubuntu-16.04'
+
+  steps:
+  - task: UsePythonVersion@0
+    inputs:
+      versionSpec: '3.x'
+      architecture: 'x64'
+
+  - template: templates/setup-ci-machine.yml
+
+  - template: templates/build-publish-aks-deploy-cli-extension.yml
+
+
+- job: 'Build_Publish_Azure_CLI_Test_SDK'
+  pool:
+    vmImage: 'Ubuntu-16.04'
+
+  steps:
+  - task: UsePythonVersion@0
+    inputs:
+      versionSpec: '3.x'
+      architecture: 'x64'
+
+  - template: templates/setup-ci-machine.yml
+
+  - template: templates/build-publish-azure-cli-test-sdk.yml
+  
+
+- job: 'Run_Style_Check'
```

Comment:
Is this for running tests? 
We are anyways running the style check later

Reply:
Sorry didn't get it.. We are running style check in this job only.

Reply:
Sorry, my bad. I thought it were two separate jobs.